### DISCUSSION TITLE
Update: Utilization of photometric redshifts in the analysis and some relevant questions

### DISCUSSION THREAD

**Discussion by SulymanHossain** [AstroBridge/BDLensing#275] [General]
We have converted the angular distances (arcsecond unit) to physical distances (kiloparsec unit) both in photometric densities and in centroid offsets, assuming H = 70 and Omega = 0.3. But we could not analytically estimate the uncertainty in physical distance from the reshift's standard deviation through the error propagation rule, as the conversion relation from redshifts to angular diameter distance seems complicated. So, we have implemented Monte Carlo random sampling to estimate the uncertainty in the physical distances, where we considered the uncertainty of redshift and angular distance. Is there any error estimation for the Hubble constant and Omega? If yes, we can also take them into account while generating the distribution of physical distances.

Another question: while calculating the physical distance of neighboring galaxies, we did it in the deflector's plane, considering the deflector's redshift. Is it similar to our initial method, where we were just considering the angular distance from the deflector? 
We don't have the individual redshift for many neighboring galaxies, and redshifts vary a lot in magnitude, so we can't correctly calculate the absolute physical distance for each neighboring galaxy.

I and @aaimtiaz were working on the above tasks, and @JobairHasan bhai also helped us brainstorm ideas.

However, as @rafee-adnan bhai suggested, we should see the photometric densities if we consider the absolute physical distance for every neighboring galaxy. At least we can do it for systems whose majority of neighboring galaxies's redshifts are known. Rafee Adnan will work on this part.

**Comment by SulymanHossain** [comment]
@ajshajib

**Comment by ajshajib** [comment]
Hi @SulymanHossain, thanks a lot for your work on this and for providing updates. For H_0 and Omega_m, you can take 70 and 0.3, respectively, without any uncertainty. It sounds great that you are using Monte Carlo sampling for error propagation.

If you don't have individual redshifts for the neighboring galaxies, I think, it's okay to assume the mean redshift of all the neighboring galaxies and the lensing galaxy for them. If you are using the absolute distances for weighting in the local number density, then I think, the angular distance is rather a more appropriate quantity to weight with, since lensing depends on angular separation. Can you remind me if you are seeing any difference in the investigated correlations based on the weighting schemes on the local number density. However, it would be great if you convert the offset between the mass and light centroids for the lensing galaxy into physical units.
